src/NuGet.Indexing/IndexingUtils.cs

```diff
@@ -28,7 +29,7 @@ public static class IndexingUtils
                 {
                     throw;
                 }
-                Trace.TraceInformation("Unable to load {0}. Exception Message : {1}", name, e.Message);
+                logger.LogInformation("Unable to load {0}. Exception Message : {1}", name, e.Message);
```

Comment:
I did not want to change this to become `LogError` (which would automatically include stack trace etc.), because the current `Trace` statement is calling `Trace*Information*`. No idea why that's the case though, happy to change it to become `logger.LogError("Unable to load {IndexName}.", name, e)`.